mitosheet/src/mito/components/taskpanes/Graph/GraphSidebar.tsx

```diff
@@ -142,44 +137,7 @@ const GraphSidebar = (props: {
         }
 
     }, [graphOutput])
-
-    /* 
-        This is the actual function responsible for loading the new
-        graph from the backend, making sure this graph is the correct
-        size.
-    */
-    const getGraphAsync = async () => {
-        // The reason that we use the mito-center-content-container instead of the graph-div is because the size of the graph div
-        // changes depending on the size of the graph. Specifically, when exiting fullscreen mode, the graph-div is wider
-        // than we actually have space for. 
-        const boundingRect: DOMRect | undefined = props.mitoContainerRef.current
-            ?.querySelector('#graph-div')
-            ?.getBoundingClientRect();
-
-        if (boundingRect !== undefined) {
-            await props.mitoAPI.editGraph(
-                graphID,
-                graphParams,
-                `${boundingRect?.height - 10}px`, // Subtract pixels from the height & width to account for padding
-                `${boundingRect?.width - 20}px`,
```

Comment:
Do we need to still be getting the height? I feel like we might -- where does this happen now?

Reply:
Oh -- I wonder if 100% just works in every case now. If it did, why would we be using the bounding rect above?